.gitignore

```diff
@@ -107,3 +107,9 @@ ENV/
 *.sublime-workspace
 
 generated/
+
+# Ignore virtual environments
+dwave_sys/
+
+# macOS system file
+.DS_Store
```

Comment:
Pro tip: keep non repo-specific ignores and non-version controlled ignores in either a parent `.gitignore`, or `.git/info/exclude`. See [docs](https://git-scm.com/docs/gitignore#:~:text=Patterns%20which,used%20instead.).

For example, move `.DS_Store` to `~/.gitignore`, and either rename the venv directory to something generic that makes sense to ignore in `~/.gitignore`, like `.env3.12`, or add `dwave_sys/` to `.git/info/exclude`.